index.jsx

```diff
@@ -1,15 +1,27 @@
 import React, {PropTypes} from 'react';
 
-function radio(name, selectedValue, onChange) {
+function radio(kwargs) {
+
   return React.createClass({
     render: function() {
+      const name = kwargs.name;
+
+      // We only want to assign these attribute to the <input />
+      // elements if they were specified explicitly.
+      const optional = {};
+      if(kwargs.hasOwnProperty("selectedValue")) {
```

Comment:
That is for sure possible, but I guess that if you are not doing that on purpose, then it should be ok. Destructuring in these functions acts as a safeguard too, to limit options that are passed through, so it would be nice to keep it this way.